apps/meteor/app/authentication/server/startup/index.js

```diff
@@ -367,6 +367,15 @@ Accounts.validateLoginAttempt(function (login) {
 		return callbacks.run('afterValidateLogin', login);
 	});
 
+	/**
+	 * Trigger the event only when the
+	 * user does login in Rocket.chat
+	 */
+	if (login.type === 'password') {
+		// App IPostUserLoggedIn event hook
+		Promise.await(Apps.triggerEvent(AppEvents.IPostUserLoggedIn, login.user));
+	}
```

Comment:
This might not be the case when the user is logging-in by using third-party services (e.g. Google, Facebook, etc.). Let's review this part